VTUL/dlp-access
# Makes description multi-valued

**JIRA Ticket**: (link) (:star:)
https://webapps.es.vt.edu/jira/browse/LIBTD-2637

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Allows users to add multiple descriptions for items and collections in the admin interface, and add multiple labels for descriptions for both collections and archives. Also updates the interface to display multiple descriptions and headings.

# What's the changes? (:star:)
**Note that descriptions must be in an array for these changes to work.
- src/components/CollectionMetadataSection.js - adds labels and filters out description from rendering in the table with all the other metadata.
- src/components/CollectionTopContent.js - updates getDescription() to work for array of descriptions.
- src/lib/MetadataRenderer.js - updates functions to work with array of descriptions and labels.
- src/lib/available_attributes.js - adds description to list of available attributes and adds lists of single and multivalued fields for archives and collections.
- src/pages/admin/ArchiveCollectionEdit/ArchiveForm.js -  replaces constants in component for single/multivalued fields
- src/pages/admin/ArchiveCollectionEdit/CollectionForm.js -  replaces constants in component for single/multivalued fields
-src/pages/admin/ArchiveCollectionEdit/ViewMetadata.js - typo fix
- src/pages/admin/DisplayedAttributesForm.js - updated to accept multiple labels for description separated by a comma. *Note: only description is fully implemented to accept multiple labels. The other multi valued fields still need addressed under another ticket.
- src/pages/archives/ArchivePage.js and src/pages/collections/CollectionsShowPage.js - updated to render multiple descriptions and labels. Also filters description from rendering in metadata table.

# How should this be tested?
-In the admin site, add multiple descriptions to an archive and/or collection. They should render on the corresponding collection and archive pages.
-In the admin site, add multiple labels to the description label. They should render as headings above each description.
-pages will also render correctly if no description or no label is available.
-Also list views on the collections browse page and search page render descriptions correctly.

# Additional Notes:
branch is multidesc

# Interested parties
Tag (@whunter whunter) interested parties

(:star:) Required fields
